src/components/response/ResponseBlock.tsx

```diff
@@ -156,10 +180,27 @@ export default function ResponseBlock({
         }
       });
 
-      setEnableNextButton((allowFailedTraining && newAttemptsUsed >= trainingAttempts) || (Object.values(correctAnswers).every((isCorrect) => isCorrect) && newAttemptsUsed <= trainingAttempts));
+      setEnableNextButton(
+        (allowFailedTraining && newAttemptsUsed >= trainingAttempts)
+        || (
+          Object.values(correctAnswers).every((isCorrect) => isCorrect)
+          && newAttemptsUsed <= trainingAttempts
+        ),
+      );
```

Comment:
```suggestion
      setEnableNextButton(
      (
          allowFailedTraining && newAttemptsUsed >= trainingAttempts
       ) || (
          Object.values(correctAnswers).every((isCorrect) => isCorrect)
          && newAttemptsUsed <= trainingAttempts
        ),
      );
```

I think might as well use the same styling syntax for both top and bottom